pkg/asset/installconfig/kubevirt/kubevirt.go

```diff
@@ -0,0 +1,130 @@
+package kubevirt
+
+import (
+	"gopkg.in/AlecAivazis/survey.v1"
+
+	"github.com/openshift/installer/pkg/types/kubevirt"
+)
+
+// Platform collects kubevirt-specific configuration.
+func Platform() (*kubevirt.Platform, error) {
+	var (
+		namespace, apiVIP, ingressVIP, networkName, storageClass string
+		err                                                      error
+	)
+
+	if namespace, err = selectNamespace(); err != nil {
+		return nil, err
+	}
+
+	if apiVIP, err = selectAPIVIP(); err != nil {
+		return nil, err
+	}
+
+	if ingressVIP, err = selectIngressVIP(); err != nil {
+		return nil, err
+	}
+
+	if networkName, err = selectNetworkName(); err != nil {
+		return nil, err
+	}
+
+	if storageClass, err = selectStorageClass(); err != nil {
```

Comment:
There might be different storage classes in cluster and there might be different storage classes between clusters.
Although there is option to use the default Storage Class (which used in case returned empty string), it was decided to let the user to control which storage class to use, as it part of the infraCluster management

Reply:
There are two different bars that must be met for fields. The first bar is whether the field is something that needs to be configured as part of installation, as opposed to configured post-installation. A field must meet that bar to be included in the install config. The second bar is whether the field has a reasonable default. For consideration to be in the user survey, the field must not have a reasonable default. The user survey is not intended to cover all ways that the user could configure the installation.

Reply:
Remove StorageClass from survey, it can be defined in install-config, default value is empty string (which in kubevirt it return the default StorageClass)